skaha/src/main/java/org/opencadc/skaha/session/GetAction.java

```diff
@@ -135,6 +188,83 @@ public void doAction() throws Exception {
             }
         }
     }
+
+    private String getNamespace() {
+        try {
+            return K8SUtil.getWorkloadNamespace();
+        } catch (Exception e) {
+            log.error(e);
+            throw new IllegalStateException("failed to get workload namespace", e);
+        }
+    }
```

Comment:
Getting the workspace is needed in many places.  Why the special method for it here?

Reply:
Deleted the method.